ci/Jenkinsfile

```diff
@@ -85,6 +85,10 @@ def testAndBuildBackend() {
 
                     stage('build-backend'){
                         sh './ci/build-scripts/build-backend.sh'
+                        sh 'cd backend && diesel migration redo'
+                        if (env.BRANCH_NAME == 'master') {
+                            sh 'cd backend && diesel migration redo -a'
```

Comment:
Isn't this logic inverted? The `redo -a` should be only done for PRs.

Any reason why you don't use the Makefile targets here?

Reply:
I was thinking to refactor all to make in one PR, but can already start I guess.